src/main/scala/cognite/spark/v1/AssetsHierarchyBuilder.scala

```diff
@@ -45,6 +45,8 @@ final case class NoRootException(
     extends Exception(message)
 final case class InvalidTreeException(message: String = s"The tree is has an invalid structure.")
     extends Exception(message)
+final case class EmptyExternalIdException(message: String = s"ExternalId cannot be an empty String.")
```

Comment:
We should include some info about which row caused this, "Found empty externalId for child with externalId = ${externalId}" (or other identifier, if we have one)

Reply:
This is only used when `externalId == ""`, so doesn't make much sense returning it, but we can return the name.

Reply:
I thought it might be a parent with external id "". But if the child itself has "" as the external id, perhaps we can display something else about the row to help the user identify it?

Reply:
For sure, we can go with the `name`. If that's `""` as well I suppose people will have to filter their data to find the culprits, but that should be fine.